plugins/X3DReader/X3DReader.py

```diff
@@ -866,13 +866,13 @@ def readIndex(node, attr):
     v = readIntArray(node, attr, [])
     chunks = []
     chunk = []
-    for i in range(len(v)):
-        if v[i] == -1:
+    for i in v:
```

Comment:
**i** is not a great variable name but this is consistent with the math style used in the file...